KsApi/models/DiscoveryParams.swift

```diff
@@ -94,25 +94,27 @@ extension DiscoveryParams: CustomStringConvertible, CustomDebugStringConvertible
 extension DiscoveryParams: Decodable {
   public static func decode(_ json: JSON) -> Decoded<DiscoveryParams> {
     let create = curry(DiscoveryParams.init)
+
     let tmp1 = create
-      <^> (json <|? "backed" >>- stringIntToBool)
+      <^> ((json <|? "backed" >>- stringIntToBool) as Decoded<Bool?>)
```

Comment:
this function took nearly 30 seconds to compile! there are a few ways we could approach this but i decided to just help out the compiler with some of the flat maps by telling it what typeof decoded value i expected